### ISSUE TITLE
Filter by tags

### ISSUE THREAD

**Issue by holocronweaver** [Fuco1/org-clock-budget#7]
Is it possible to filter the budget report by tag? If not, would this be difficult to implement?

**Comment by philohistoria** [comment]
I agree and think that filtering by tags will be a highly useful function.

**Comment by Fuco1** [comment]
I think if I generate the rows in a format similar enough to the agenda we should be able to reuse those commands, but currently it's not possible.
